content/docs/forms.md

```diff
@@ -105,46 +105,46 @@ class EssayForm extends React.Component {
   }
 
   handleSubmit(event) {
-    alert('An essay was submitted: ' + this.state.value);
+    alert('Uma dissertação foi enviada: ' + this.state.value);
     event.preventDefault();
   }
 
   render() {
     return (
       <form onSubmit={this.handleSubmit}>
         <label>
-          Essay:
+          Dissertação:
           <textarea value={this.state.value} onChange={this.handleChange} />
         </label>
-        <input type="submit" value="Submit" />
+        <input type="submit" value="Enviar" />
       </form>
     );
   }
 }
 ```
 
-Notice that `this.state.value` is initialized in the constructor, so that the text area starts off with some text in it.
+Observe que `this.state.value` é inicializado no construtor, para que o textarea comece com algum texto.
 
-## The select Tag {#the-select-tag}
+## Tag select {#the-select-tag}
 
-In HTML, `<select>` creates a drop-down list. For example, this HTML creates a drop-down list of flavors:
+Em HTML, `<select>` cria uma lista suspensa. Por exemplo, esse HTML cria uma lista suspensa de sabores:
 
 ```html
 <select>
-  <option value="grapefruit">Grapefruit</option>
-  <option value="lime">Lime</option>
-  <option selected value="coconut">Coconut</option>
-  <option value="mango">Mango</option>
+  <option value="laranja">Laranja</option>
+  <option value="limão">Limão</option>
+  <option selected value="coco">Coco</option>
+  <option value="manga">Manga</option>
```

Comment:
I don't think the examples should be translated

Reply:
@cstpk 

Nesse caso são só strings, você acha que prejudica muito?

Eu pensei em pedir pra ele reverter, mas a primeira sugestão do [styleguide](https://github.com/reactjs/pt-BR.reactjs.org/blob/master/GLOSSARY.md) é:

> Leave code blocks untranslated -- It is usually OK to translate comments and strings inside them.

O que acha?

Reply:
Eu não acho que prejudica, pelo contrário, ajuda no entendimento dos exemplos para pessoas que não sabem inglês.

Reply:
@httpiago acho q pode manter assim, ou talvez deixar o conteúdo da option em português, mas o do atributo `value` deixaria em inglês, por exemplo, quando estou programando meu backend sempre é todo em inglês, apenas o front trada outros idiomas. Não sei se ajudei :see_no_evil:

Reply:
```suggestion
 <option value="laranja">Laranja</option>
  <option value="limao">Limão</option>
  <option selected value="coco">Coco</option>
  <option value="manga">Manga</option>
```

Reply:
@jhonmike precisa estar os values em português, porém sem acentuação.

Reply:
@glaucia86 @jhonmike @httpiago @cstpk 
Oi Pessoal, nesse caso em específico eu estou achando essa tradução das strings complicada. 

O que acordamos em relação a strings seria que "de forma geral", poderiam ser traduzidas, mas utilizando o bom senso para julgar onde não devem. Nesse caso aqui eu acho que adicionamos uma carga cognitiva muito grande em cima do programador pois criamos muitas possibilidades de confusão:

1.  Pode traduzir, mas no value não pode ter acento.
2. Se o value for comparado mas à frente com qualquer string, essa string vai ter que ser traduzida também, caso contrário estaríamos introduzindo um erro de lógica/comparação
3. Se qqr um dos valores não for traduzido, por acidente, estaríamos também introduzindo um erro, pois se o usuário copiar e colar o código e tiver uma string sem tradução, dependendo de como ela estiver sendo utilizada, poderá quebrar o código.
4. Se houver reutilização do código em qualquer outra parte da documentação, há a possibilidade dele não ter sido traduzido, e confundir o usuário.

Resumindo, eu acho que strings "inocentes" tipo labels ou alguma coisa mais simples que só faça parte da interface são ok para traduzir, mas é arriscado traduzir quando a string faz parte da lógica, cria-se muitos pontos frágeis para introdução de erros.

Enfim, essa é minha visão. Vejam aí se faz sentido!